src/plugins/column-filter/column-filter-by.component.ts

```diff
@@ -10,12 +10,23 @@ import { ColumnModel } from 'ng2-qgrid/core/column-type/column.model';
 export class ColumnFilterByComponent extends PluginComponent {
 	@Input() by: Set<string>;
 	@Input() column: ColumnModel;
+	@Input() byBlanks: boolean;
+
+	@Output() byBlanksChange = new EventEmitter();
 
 	constructor(@Optional() root: RootService) {
 		super(root);
 	}
 
+	get isBlanks() {
+		return this.byBlanks;
+	}
+
 	remove(item: string): void {
-		this.by.delete(item);
+		if (item) {
```

Comment:
make for byBlanks another method removeByBlanks, because in theory item can be an empty string